node/src/admin_client.ts

```diff
@@ -67,4 +71,22 @@ export class AdminClient {
       });
     });
   }
+
+  /**
+   * Send check paused request to a host.
+   * @returns {boolean} paused or not
+   */
+  async checkPaused(): Promise<boolean> {
```

Comment:
Nothing wrong with the code at all but just a FYI you can write this code like this as well.  Reduces the need for the outer Promise constructor.  No right or wrong way here though. :) 

```Typescript
async checkPaused(): Promise<boolean> {
    const client = this._client;

      client.checkPaused(new Empty(), (e: unknown, response: any) => {
        if (e !== null) {
          return Promise.reject(e);
        } else {
          return Promise.resolve(response.getPaused());
        }
      });
  }
```

Reply:
> Nothing wrong with the code at all but just a FYI you can write this code like this as well. Reduces the need for the outer Promise constructor. No right or wrong way here though. :)
> 
> ```ts
> async checkPaused(): Promise<boolean> {
>     const client = this._client;
> 
>       client.checkPaused(new Empty(), (e: unknown, response: any) => {
>         if (e !== null) {
>           return Promise.reject(e);
>         } else {
>           return Promise.resolve(response.getPaused());
>         }
>       });
>   }
> ```

Wow. I didn't know that.
It's much clear.
I will revise that part.

Reply:
after discussing with Yves, we still need to wrap the callback-styled functions by the Promise construction.

But, after I checked the community's convention, I revised the part in
https://github.com/scalar-labs/scalar-admin/pull/16/commits/fa113a30cd15354fcc09dbd497863bb1c4105de3

It is to use
```
if (e) reject(e)
resolve()
```
to replace
```
if (e !== null) {
    reject(e)
} else {
    resolve()
}
```

Since it's a pattern in the community, so I won't ask for a review again to save time.